### DISCUSSION TITLE
\\u0000 cannot be converted to text.

### DISCUSSION THREAD

**Discussion by indahud** [supabase/supabase#2264] [Questions]
I don't understand why I am getting this error. I am trying to insert some data. I don't have anything like `\\u0000` in my data. 
Here's error code and message.
```JavaScript
details: "\\u0000 cannot be converted to text."
code: "22P05"
message: "unsupported Unicode escape sequence"
```

Any ideas how I can resolve this issue? I am not able to understand what is causing the error. 
```Javascript
const InsertData = async (req, res) => {
  try {
    const { data, error } = await supabase.from('testdb').insert([])  
    return res.status(200).json({
      messgae: "Data Inserted",
      data: data,
      error: error,
    });
  } catch (error) {
    console.log(error);
  }
}
```

**Comment by faizanraso** [comment]
I believe \\u0000 is unicode for null, and in the above example it seems like you are inserting null into the db which may be causing that error.

**Reply by tilensavnik** [reply]
thanks!
